packages/buttons/src/elements/Button.tsx

```diff
@@ -77,5 +77,21 @@ Button.defaultProps = {
   size: 'medium'
 };
 
+interface IIconProps {
+  isRotated?: boolean;
+  children: any;
+}
+
+const StartIcon = (props: IIconProps) => <StyledIcon position="start" {...props} />;
+const EndIcon = (props: IIconProps) => <StyledIcon position="end" {...props} />;
```

Comment:
Should these elements use `React.forwardRef`?

Reply:
Testing shows that we can't reference function components due to the child cloning: https://github.com/zendeskgarden/react-components/blob/992c41ff83f4f6349bd491b6cdbe3dc9373b0fae/packages/buttons/src/styled/StyledIcon.ts#L19-L21

I think we'll expect consumers to put the `ref` on the `<Button>` and drill into the DOM if necessary.